### ISSUE TITLE
Traffic Graph tab enhancement

### ISSUE THREAD

**Issue by loganhz** [rancher/rancher#20896]
Use IFrame to move the Kiali Graph into Rancher UI.

**Comment by davidnuzik** [comment]
Version: master-head (7/26/19)

This change simply creates an iframe now for the "Traffic Graph" tab. We have an iframe of what we see in Kiali:

![image](https://user-images.githubusercontent.com/45179589/61981542-d5f0a900-afae-11e9-870a-78a4c66947aa.png)
![image](https://user-images.githubusercontent.com/45179589/61981552-dab55d00-afae-11e9-883a-d38ddd7c4962.png)

This let's us leverage Kiali UI features such as pictured below (traffic animation and weight)
![image](https://user-images.githubusercontent.com/45179589/61981637-0e908280-afaf-11e9-9cc8-0a2dffe9ffce.png)

I tested each option in the UI as carefully as I could -- essentially smoke testing all the different views and options. I made sure any pop-ups (window overlays like the Legend) work as expected. All in all it's essentially an iframe and it works as expected.

I tried to break things and wasn't able to identify any issues. Rancher is showing the iframe in a manner I expect and that makes sense to me. I was not able to break anything in the Kiali UI (which would be a Kiali issue anyway).
